app/views/detainees/_form.html.slim

```diff
@@ -14,8 +14,8 @@
 
   .form-wrapper
     = form_for form, url: submit_path, method: method do |f|
-      label.form-label Prison number
-      p.prison_number = escort.prison_number
+      label.form-label Number
```

Comment:
I just noticed this and it's not correct, there is still a prison number and a pnc number where is from prison or police. Can you please revert these changes?